diff -urN bmpx-0.40.1/data/desktop/bmp-2.0.desktop.in bmpx-0.40.1.new/data/desktop/bmp-2.0.desktop.in
--- bmpx-0.40.1/data/desktop/bmp-2.0.desktop.in	2007-08-11 20:41:55.000000000 +0200
+++ bmpx-0.40.1.new/data/desktop/bmp-2.0.desktop.in	2007-08-13 16:41:50.000000000 +0200
@@ -15,10 +15,9 @@
 Icon=bmpx
 #MimeType=audio/x-scpls;audio/x-mpegurl;audio/mpegurl;audio/mp3;audio/x-mp3;audio/mpeg;audio/x-mpeg;audio/x-wav;application/x-ogg;
 #Actions=Enqueue;Play;
-Categories=Application;GTK;AudioVideo;Player;
+Categories=GTK;AudioVideo;Player;
 Terminal=false
 Type=Application
-Encoding=UTF-8
 StartupNotify=false
 
 #[Desktop Action Enqueue]
diff -urN bmpx-0.40.1/data/desktop/bmp-2.0-offline.desktop.in bmpx-0.40.1.new/data/desktop/bmp-2.0-offline.desktop.in
--- bmpx-0.40.1/data/desktop/bmp-2.0-offline.desktop.in	2007-08-11 20:41:55.000000000 +0200
+++ bmpx-0.40.1.new/data/desktop/bmp-2.0-offline.desktop.in	2007-08-13 16:41:35.000000000 +0200
@@ -15,10 +15,9 @@
 Icon=bmpx
 #MimeType=audio/x-scpls;audio/x-mpegurl;audio/mpegurl;audio/mp3;audio/x-mp3;audio/mpeg;audio/x-mpeg;audio/x-wav;application/x-ogg;
 #Actions=Enqueue;Play;
-Categories=Application;GTK;AudioVideo;Player;
+Categories=GTK;AudioVideo;Player;
 Terminal=false
 Type=Application
-Encoding=UTF-8
 StartupNotify=false
 
 #[Desktop Action Enqueue]
diff -urN bmpx-0.40.1/data/desktop/bmp-enqueue-2.0.desktop.in bmpx-0.40.1.new/data/desktop/bmp-enqueue-2.0.desktop.in
--- bmpx-0.40.1/data/desktop/bmp-enqueue-2.0.desktop.in	2007-08-11 20:41:55.000000000 +0200
+++ bmpx-0.40.1.new/data/desktop/bmp-enqueue-2.0.desktop.in	2007-08-13 16:42:34.000000000 +0200
@@ -11,10 +11,8 @@
 Name[dk]=BMPx: Tilføj til afspilningsliste
 Exec=beep-media-player-2 --add-files %F
 Icon=bmpx
-MimeType=audio/mp3;audio/x-mp3;audio/mpeg;audio/x-mpeg;audio/x-wav;application/x-ogg;application/ogg;audio/x-m4a
+MimeType=audio/mp3;audio/x-mp3;audio/mpeg;audio/x-mpeg;audio/x-wav;application/x-ogg;application/ogg;audio/x-m4a;
 Terminal=false
 Type=Application
 NoDisplay=true
 X-KDE-StartupNotify=false
-Encoding=UTF-8
-
diff -urN bmpx-0.40.1/data/desktop/bmp-play-2.0.desktop.in bmpx-0.40.1.new/data/desktop/bmp-play-2.0.desktop.in
--- bmpx-0.40.1/data/desktop/bmp-play-2.0.desktop.in	2007-08-11 20:41:55.000000000 +0200
+++ bmpx-0.40.1.new/data/desktop/bmp-play-2.0.desktop.in	2007-08-13 16:43:01.000000000 +0200
@@ -12,10 +12,9 @@
 Comment=BMP 2
 Exec=beep-media-player-2 --add-files --playback %F
 Icon=bmpx
-MimeType=audio/mp3;audio/x-mp3;audio/mpeg;audio/x-mpeg;audio/x-wav;application/x-ogg;application/ogg;audio/x-m4a
+MimeType=audio/mp3;audio/x-mp3;audio/mpeg;audio/x-mpeg;audio/x-wav;application/x-ogg;application/ogg;audio/x-m4a;
 Terminal=false
 Type=Application
 NoDisplay=true
 X-KDE-StartupNotify=false
-Encoding=UTF-8
 
