diff -up bmpx-0.40.13/src/sanity.cc.constcastfix bmpx-0.40.13/src/sanity.cc
--- bmpx-0.40.13/src/sanity.cc.constcastfix	2007-10-18 19:17:08.000000000 +0200
+++ bmpx-0.40.13/src/sanity.cc	2007-11-29 16:05:38.000000000 +0100
@@ -72,9 +72,9 @@ C_FUNCTION
 gchar *
 sanity_check_gtk (void)
 {
-  return gtk_check_version (GTK_REQUIRED_MAJOR_VER,
-                            GTK_REQUIRED_MINOR_VER,
-                            GTK_REQUIRED_MICRO_VER);
+  return const_cast<gchar *>(gtk_check_version (GTK_REQUIRED_MAJOR_VER,
+						GTK_REQUIRED_MINOR_VER,
+						GTK_REQUIRED_MICRO_VER));
 }
 
 static gchar *
