diff -urN bmpx-0.40.13.org/configure.ac bmpx-0.40.13/configure.ac
--- bmpx-0.40.13.org/configure.ac	2007-10-18 19:17:17.000000000 +0200
+++ bmpx-0.40.13/configure.ac	2007-11-26 18:17:26.398334212 +0100
@@ -406,7 +406,7 @@
     if test "$have_sid" = "yes"; then
         AC_DEFINE([HAVE_SID], [1], [have sid])
         SID_INCLUDES="-I$sid_prefix/sidplay"
-        SID_LIBS="-L$sid_prefix/lib -lsidplay"
+        SID_LIBS="-lsidplay"
     else
         AC_MSG_ERROR([libsidplay 1.x not found!])
     fi
@@ -586,7 +586,7 @@
           else
             CDPARANOIA_INCLUDES="-I$cdparanoia_prefix/include"
           fi
-          CDPARANOIA_LIBS="-L$cdparanoia_prefix/lib -lcdda_interface"
+          CDPARANOIA_LIBS="-lcdda_interface"
   else
           AC_MSG_ERROR([cdparanoia not found!])
   fi
diff -urN bmpx-0.40.13.org/m4/boost_base.m4 bmpx-0.40.13/m4/boost_base.m4
--- bmpx-0.40.13.org/m4/boost_base.m4	2007-10-18 19:17:06.000000000 +0200
+++ bmpx-0.40.13/m4/boost_base.m4	2007-11-26 18:14:17.614634080 +0100
@@ -67,12 +67,10 @@
 	dnl this location ist chosen if boost libraries are installed with the --layout=system option
 	dnl or if you install boost with RPM
 	if test "$ac_boost_path" != ""; then
-		BOOST_LDFLAGS="-L$ac_boost_path/lib"
 		BOOST_CPPFLAGS="-I$ac_boost_path/include"
 	else
 		for ac_boost_path_tmp in /usr /usr/local /opt /opt/local /usr/lib64 /usr/local/lib64; do
 			if test -d "$ac_boost_path_tmp/include/boost" && test -r "$ac_boost_path_tmp/include/boost"; then
-				BOOST_LDFLAGS="-L$ac_boost_path_tmp/lib"
 				BOOST_CPPFLAGS="-I$ac_boost_path_tmp/include"
 				break;
 			fi
@@ -111,7 +109,6 @@
 	if test "x$succeeded" != "xyes"; then
 		_version=0
 		if test "$ac_boost_path" != ""; then
-               		BOOST_LDFLAGS="-L$ac_boost_path/lib"
 			if test -d "$ac_boost_path" && test -r "$ac_boost_path"; then
 				for i in `ls -d $ac_boost_path/include/boost-* 2>/dev/null`; do
 					_version_tmp=`echo $i | sed "s#$ac_boost_path##" | sed 's/\/include\/boost-//' | sed 's/_/./'`
@@ -139,7 +136,6 @@
 
 			VERSION_UNDERSCORE=`echo $_version | sed 's/\./_/'`
 			BOOST_CPPFLAGS="-I$best_path/include/boost-$VERSION_UNDERSCORE"
-			BOOST_LDFLAGS="-L$best_path/lib"
 
 	    		if test "x$BOOST_ROOT" != "x"; then
 				if test -d "$BOOST_ROOT" && test -r "$BOOST_ROOT" && test -d "$BOOST_ROOT/stage/lib" && test -r "$BOOST_ROOT/stage/lib"; then
